content/learn/book/next-steps/_index.md

```diff
@@ -12,7 +12,7 @@ You have reached the end of The Bevy Book! And unfortunately, we haven't even sc
 * **[The Bevy Examples](https://github.com/bevyengine/bevy/tree/latest/examples#examples)**: We create an example for every major Bevy feature. This is currently the best way to learn Bevy's features and how to use them.
   * **[The Bevy Web Examples](https://bevyengine.org/examples)**: We also try to make most of these examples available directly in your browser.
 * **[Breakout](https://github.com/bevyengine/bevy/blob/latest/examples/games/breakout.rs)**: A small "breakout" clone that illustrates what a real Bevy game looks like.
-* **[Bevy API Docs](https://docs.rs/bevy)**: Bevy's Rust API documentation.
+* **[Bevy API Docs](https://docs.rs/bevy)**: Bevy's Rust API reference documentation which includes both auto-generated entries and hand-written explanations.
```

Comment:
```suggestion
* **[Bevy API Docs](https://docs.rs/bevy)**: Documentation for each type and trait used in Bevy.
```

I don't think that including either "Rust" or "API" is useful here, but it's important to communicate that it's granular.